lib/ruumba/rake_task.rb

```diff
@@ -15,8 +15,9 @@ class RakeTask < Rake::TaskLib
     def initialize(*args, &block)
       @name = args.shift || :ruumba
       @dir  = []
+      @options = nil
 
-      desc 'Run RuboCop on ERB files' unless ::Rake.application.last_comment
+      desc 'Run RuboCop on ERB files' # unless ::Rake.application.last_comment
```

Comment:
@ybart Feel free to remove the commented-out `unless ::Rake.application.last_comment`